nmrfx-processor/src/main/resources/pyproc.py

```diff
@@ -3869,7 +3870,8 @@ def genScript(arrayed=False):
         if fCoef != None and fCoef == 'sep':
             script += "MAG()\n"
         else:
-            script += 'PHASE(ph0=0.0,ph1=0.0)\n'
+            phases = NMRDataUtil.getPhases(fidInfo.fidObj, iDim - 1);
+            script += 'PHASE(ph0='+str(round(phases[0],1))+',ph1='+str(round(phases[1]))+')\n'
```

Comment:
this will now try to use phases from the files, but if not available it will use autophase, is this okay?

Reply:
Now getPhases will return 0, 0 if the phases aren't set and the dimension is greater than 0